nerfstudio/models/splatfacto.py

```diff
@@ -686,6 +700,27 @@ def _get_background_color(self):
             raise ValueError(f"Unknown background color {self.config.background_color}")
         return background
 
+    def _apply_bilateral_grid(self, rgb: torch.Tensor, cam_idx: int, H: int, W: int) -> torch.Tensor:
+        if cam_idx != 0:
+            # make xy grid
+            grid_y, grid_x = torch.meshgrid(
+                torch.linspace(0, 1.0, H),
+                torch.linspace(0, 1.0, W),
+                indexing="ij",
+            )
+            grid_xy = torch.stack([grid_x, grid_y], dim=-1).unsqueeze(0).to(self.device)
+
+            out = slice(
+                bil_grids=self.bil_grids,
+                rgb=rgb,
+                xy=grid_xy,
+                grid_idx=torch.tensor(cam_idx, device=self.device, dtype=torch.long),
+            )
+            return out["rgb"]
+        else:
+            # do not apply bilateral grid to the first camera
```

Comment:
Why is the bilateral grid not applied to the first camera?

Reply:
I just let the first camera work as the reference, but this may hurt if the 1st photo is overexposured. I think I should make it also be applied.